messages/jsonschema/scripts/templates/ruby.rb.erb

```diff
@@ -24,6 +24,29 @@ module Cucumber
       attr_reader :<%= underscore(property_name) %>
```

Comment:
I love that we're keeping the underscore / camelCase stuff out of the runtime code!